gtda/mapper/utils/_visualization.py

```diff
@@ -248,22 +244,30 @@ def _validate_color_kwargs(graph, data, color_data, color_features,
     elif color_features_kind == "callable":
         color_data_transformed = color_features(color_data)
         # If outcome is a pandas dataframe, save column names before converting
-        # to numpy array
+        # to numpy array. If a pandas series, just convert to numpy
         if hasattr(color_data_transformed, "columns"):
```

Comment:
What about using the `is_color_data_dataframe` instead?

Reply:
Because here we are checking that the *transformed* data is a dataframe.  As an example, pd.get_dummies will work on a numpy array but will return a df.

Reply:
Right, ok, thank you